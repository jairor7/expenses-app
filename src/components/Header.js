import React from "react";
import { NavLink } from "react-router-dom";

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <nav>
      <NavLink to={"/"}>Dashboard</NavLink>
      <NavLink to={"/createExpense"}>Create Expense</NavLink>
      <NavLink to={"/contact"}>Contact</NavLink>
    </nav>
  </header>
);

export default Header;
