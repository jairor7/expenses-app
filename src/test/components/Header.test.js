import Header from "../../components/Header";
import React from "react";
import toJSON from "enzyme-to-json";
import { shallow } from "enzyme";

test("should render Header correctly", () => {
  const wrapper = shallow(<Header />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
