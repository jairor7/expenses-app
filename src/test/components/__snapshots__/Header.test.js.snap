// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <nav>
    <NavLink
      to="/"
    >
      Dashboard
    </NavLink>
    <NavLink
      to="/createExpense"
    >
      Create Expense
    </NavLink>
    <NavLink
      to="/contact"
    >
      Contact
    </NavLink>
  </nav>
</header>
`;
